packages/interpretations/src/components/interpretations/Interpretation.js

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
-import ThumbUpIcon from '@material-ui/icons/ThumbUp';
 import i18n from '@dhis2/d2-i18n';
 import SharingDialog from '@dhis2/d2-ui-sharing-dialog';
+import { withStyles } from '@material-ui/core/styles';
 import some from 'lodash/fp/some';
 import InterpretationComments from './InterpretationComments';
 import InterpretationDialog from './InterpretationDialog';
```

Comment:
InterpretationDialog file has been deleted.

Reply:
I've re-created that file and added it back. I assume the symlink were still present and pointed to the inode which resided in my trash folder, so i didnt get any errors.